shared/types/dataModels.ts

```diff
@@ -0,0 +1,258 @@
+import { Operation } from '../external/models';
+import { ZapiPropertyReferencingASchema } from '../models/zapi_property_referencing_a_schema';
+import { ZapiSchemaAllof } from '../models/zapi_schema__allof';
+import { ZapiSchemaAnyof } from '../models/zapi_schema__anyof';
+import { ZapiSchemaArray } from '../models/zapi_schema__array';
+import { ZapiSchemaBoolean } from '../models/zapi_schema__boolean';
+import { ZapiSchemaInteger } from '../models/zapi_schema__integer';
+import { ZapiSchemaNumber } from '../models/zapi_schema__number';
+import { ZapiSchemaObject } from '../models/zapi_schema__object';
+import { ZapiSchemaOneof } from '../models/zapi_schema__oneof';
+import { ZapiSchemaString } from '../models/zapi_schema__string';
+
+export interface IPreprocessedData {
+    readonly zapiSpecificationCodename: string,
+    readonly items: any,
```

Comment:
What about:
```suggestion
    readonly items: unknown,
```
It would be semantically more correct, wouldn't it? 😇